### PR TITLE
i18n: Clarify messages emitted when clicking things to copy them

### LINKED ISSUE #136: Copy text
On details page, if you click on any address it automatically copies that address. The pop up text for this is confusing -- it says "xxx copied". Would recommend changing this to "Copied to Clipboard"
Screen Shot 2021-09-08 at 9 08 50 PM

### PR DESCRIPTION
**Pull request by tjanez**
Fixes #136.

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -45,8 +45,8 @@
     "txHash": "Transaction Hash",
     "receiveTitle": "Receive",
     "walletAddress": "Wallet Address",
-    "copyAddress": "Copy Address",
-    "copySuccess": "Copied",
+    "copyAddress": "Address Copied to Clipboard",
```
Comment (pro-wh) [inline src/i18n/en.json:48]: where is this used?

**Comment by pro-wh** [comment]
what did the "xxx" mean in the issue?

File: src/i18n/en.json (hunk 1 above)
Comment (tjanez) [reply src/i18n/en.json:48]: If you go to the Staking tab, open the Validators tab, click on a validator to get to the Validator Info page and then click on its address or the "Address" label.

Comment (pro-wh) [reply src/i18n/en.json:48]: image

this is from before. what's it like after the change?

Comment (pro-wh) [reply src/i18n/en.json:48]: oops wrong screenshot, will update

**Comment by tjanez** [comment]
> what did the "xxx" mean in the issue?

That the "Copied"/"Copied to Clipboard" message is always prefixed with the transaction's field that has been copied, e.g.:
- "Amont Copied to Clipboard"
- "To Copied to Clipboard"
- "From Copied to Clipboard"
- "Fee Copied to Clipboard"
...

I would leave the prefix logic as it is for the time being and just improve message by replacing "Copied" with "Copied to Clipboard".

File: src/i18n/en.json (hunk 1 above)
Comment (tjanez) [reply src/i18n/en.json:48]: I don't have the screenshot but the message says: "Address Copied to Clipboard"

**Comment by pro-wh** [comment]
"copied" alone was fine imo. adding "to clipboard" I suppose doesn't worsen it though, so fine.

what about the `copyAddress` string? I couldn't find any uses of that

File: src/i18n/en.json (hunk 1 above)
Comment (tjanez) [reply src/i18n/en.json:48]: Oh, nevermind, the `"copyAddress"` i18n string is actually never used.

I'll remove it.

**Review by pro-wh — APPROVED** [review]
ok if you prefer it this way

**Comment by tjanez** [comment]
> what about the `copyAddress` string? I couldn't find any uses of that

Me neither. Let's remove it: https://github.com/oasisprotocol/oasis-wallet-ext/pull/142#discussion_r705751542.

**Pull request merged**